### DISCUSSION TITLE
Prusa Slicer infill anchor

### DISCUSSION THREAD

**Discussion by errhoss** [OrcaSlicer/OrcaSlicer#122] [Ideas]
For a print I'm doing I need to set the infill anchor to 0 and I don't seem to be able to do that in bambu studio. If there was a way to expose the setting from prusa slicer that would be amazing!

**Comment by errhoss** [comment]
![Screenshot_20230105_082552](https://user-images.githubusercontent.com/69829229/210917420-98a8fd75-55bf-4a2a-bcaa-614bf030b318.png)
With infill anchor

**Comment by errhoss** [comment]
![Screenshot_20230105_082517](https://user-images.githubusercontent.com/69829229/210917453-50f3f42b-9dc7-43df-bc7f-99f49198cb69.png)
Without infill anchor

**Comment by kyzxyz** [comment]
I'm also trying something with exposed infills and it would be good if this option was available like in Prusa/Superslicer.

![image](https://user-images.githubusercontent.com/127074349/223020086-f26495fe-9f42-46fa-97fc-6e89303bf4da.png)

**Comment by wociscz** [comment]
Slicing the HeadAmame phones with prusaslicer/superslicer or bambustudio I had no luck - there are always "holes" on the surface underneath (or over) the infill body part touching the shell part - not only the infill anchor problem. Check your sliced model - you'll find the holes. Only possible way to properly slice it was with Cura.
And tried a lot of trickery/settings - no luck. Try Cura for this.

**Comment by millpreet** [comment]
anyone solve this?
